tests/regression/87-once/04-thread.c

```diff
@@ -0,0 +1,58 @@
+// PARAM: --set ana.activated[+] pthreadOnce
+#include <pthread.h>
+#include <stdio.h>
+
+int g;
+int h;
+int i;
+pthread_once_t once = PTHREAD_ONCE_INIT;
+pthread_once_t i_once = PTHREAD_ONCE_INIT;
+pthread_mutex_t mut;
+
+void *t_other(void* arg) {
+  g = 17; //RACE
+
+  pthread_mutex_lock(&mut);
+  i = 7; //NORACE
+  pthread_mutex_unlock(&mut);
+}
+
+void nesting() {
+  h = 5; //NORACE
+}
+
+void fun() {
+  // Even though this is only called inside the once, the accesses in the new thread and the accesses here can happen in parallel
+  // Checks that active is not passed to the created thread, but seen is passed
+  pthread_t tid = pthread_create(&tid, NULL, t_other, NULL);
+  g = 42; //RACE
```

Comment:
If there's an actual race possible, not just our imprecision, then this should be `RACE!` here and on the other side.